packages/expo-modules-core/android/build.gradle

```diff
@@ -49,7 +49,8 @@ def isAndroidTest = {
   return false
 }.call()
 
-def downloadsDir = new File("$buildDir/downloads")
+def customDownloadsDir = System.getenv("REACT_NATIVE_DOWNLOADS_DIR")
```

Comment:
That's nice improvement, but maybe we should also set this to a custom dir on the CI and cache this dir?